CHANGELOG.md

```diff
@@ -15,7 +15,7 @@
 - **Breaking change** Combines `MoyaPath` and `MoyaTarget` protocols.
 - **Breaking change** Renames `Moya/Reactive` subspec to `Moya/ReactiveCocoa`.
 - **Breaking change** Removes `stubResponses` from initializer; replaced with new stubbing behavior `.NoStubbing`. Added class methods to `MoyaProvider` to provide defaults, while allowing users to still change stubbing behaviour on a per-request basis.
-- **Breaking change** Redefines types of `DefaultEndpointMapping` and `DefaultEnpointResolution` class functions on `MoyaProvider`. You no longer invoke these functions to return a closure, rather, you reference the functions themselves _as_ closures.
```

Comment:
Since we're going to make this a change to `3.0.0`, we should keep this line the same so people still using 2.x can rely on the changelog.